invite/templates/invite/invite.html

```diff
@@ -122,14 +127,58 @@
 
         // Register the keyup event handler so the search boxes work
         $("#perms-search, #groups-search").keyup(searchOptions);
+
+        // With a new form, make sure only the first form row has the "add" button
+        $("#invitees button#add:gt(0)").remove();
+
+        // Check to see if the username or email address is already in use, and display error if so
+        function validateField() {
+            var xhttp = new XMLHttpRequest();
+            var $input = $(this);
+            var $row = $input.closest(".control-group");
+            var type = $input.attr("placeholder").toLowerCase();
+            var usernameError = $("<li></li>").text("Username: Username taken, choose another");
+            var emailError = $("<li></li>").text("Email: The email provided already belongs to a user");
+            var typeToError = {"email":emailError, "username":usernameError};
+            var error = typeToError[type];
+            xhttp.onreadystatechange = function() {
+                if (this.readyState === 4 && this.status === 200) {
+                    var taken = JSON.parse(this.responseText).taken;
+                    if (taken === true) {
+                        var errorDiv = $("<div class='alert alert-error'><button type='button' class='close' data-dismiss='alert'>×</button></div>");
```

Comment:
simply for consistency. Before my changes, when you submitted a form with, for instance, an invalid email, you'd get errors just like the ones that are showing with this PR, but they can be dismissed. So this is _just_ for consistency when you have those errors showing up. Didn't want some to be able to be dismissed while others that look identical couldn't.

Reply:
Are there any of the dismissible errors that it is ok to ignore? It seems weird to me to allow a user to `x` out an error message as if it is ok for them to continue on without changing their bad input error...especially since they automatically go away when the user fixes it... Though maybe this was a specific UI feature someone wanted initially? We could get rid of the `x` button on all of them rather than keeping it on all of them, though if there is a reason we want to keep it--I will drop it the topic. :grinning:

Reply:
Well I can see the original reason being that those errors wouldn't go away even when fixed. The automatic dismissal of those is new to this PR. However, that being the case, I think we can now remove the dismissal button from all errors, as you suggested.

Reply:
I think that is a good idea. I like the automatic dismissal functionality you added and removes the need for a dismissal button.